fetch.bs

```diff
@@ -2953,7 +2953,7 @@ given a <a for=/>request</a> <var>request</var>, run these steps:
 
   <ol>
    <li>
-    <p>Switch on <var>request</var>'s <a for=request>referrer policy</a>:
+    <p>If <var>request</var>'s <a for=request>method</a> is not "<code>cors</code>", then switch on <var>request</var>'s <a for=request>referrer policy</a>:
```

Comment:
You want **mode** here, not method.

Reply:
Thanks, good point.

This could be improved by having the if and switch be different steps, but I am not sure what the best way of doing this is.

Reply:
You would have to create a nested list for the switch, but I'm not sure that's worth it? Feel free to ping me on Matrix if something is unclear.

Reply:
No, I just looked at the output again and think it's fine.